pages/contribute/github_way.md

```diff
@@ -44,12 +45,20 @@ summary: This is a test summary
   * Click on “Commit changes”.
   
 ## Request a revision
-1. Follow step 7 (see above)
-2. Select "Pull Request" instead of "Draft Pull Request". Editors will be notified and they will check your changes.
+1. Go to [RDM toolkit website](https://elixir-europe.github.io/rdm-toolkit/) and click on the "GitHub" button on the top navigation bar. It will take you in the GitHub repository of our RDM toolkit.
```

Comment:
```suggestion
1. Go to [RDM toolkit website](https://elixir-europe.github.io/rdm-toolkit/) and click on the "GitHub" button on the top navigation bar. It will take you to the GitHub repository of our RDM toolkit.
```